content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst

```diff
@@ -1,54 +1,89 @@
 
 Configuration
 =============
 
-In the *Inventory* app, go to :menuselection:`Configuration --> Settings` and activate the
-*Multi-Step Routes*. By doing so, the *Storage Locations* will be automatically activated.
+Begin by navigating to :menuselection:`Inventory --> Configuration --> Settings`, then activate the
+:guilabel:`Multi-Step Routes` checkbox under the :guilabel:`Warehouse` heading. By doing so, the
+:guilabel:`Storage Locations` setting is automatically enabled as well. Finally, click
+:guilabel:`Save`.
 
-.. image:: putaway/putaw1.png
+.. image:: putaway/multi-step-routes.png
    :align: center
+   :alt: Check multi-step routes box
 
-Setting up a Putaway Rule
-=========================
+Create a putaway rule
+=====================
 
-In some cases, like for a retail shop storing vegetables and fruits, we have to store products in
-different locations to maintain product quality.
+To manage where specific products are routed for storage, navigate to :menuselection:`Inventory -->
+Configuration --> Putaway Rules`. Then, click on :guilabel:`Create` to configure a new putaway rule.
+Choose the :guilabel:`Product` and/or :guilabel:`Product Category` that the rule will affect. Set
+:guilabel:`When product arrives in` as the :ref:`location<inventory/warehouse_location/definition>`
+where the rule will be triggered and :guilabel:`Store to` as the location where products affected by
+the rule will be stored. Finally, click :guilabel:`Save`.
 
-Let’s suppose there are one warehouse location *WH/Stock* and two sub-locations
-*WH/Stock/Vegetables* and *WH/Stock/Fruits*.
-
-To manage those locations, we will create putaway rules. To do so, open the *Inventory* app and go
-to :menuselection:`Configuration --> Putaway Rules`. Then, click on create and configure your first
-rule indicating the main location the product will enter before being redirected to the right
-location.
+.. image:: putaway/create-putaway-rules.png
+   :align: center
+   :alt: Create putaway rules for apples and carrots
 
 .. note::
-   The putaway rules can be defined either per product or per product category.
+  It is also possible to create and manage putaway rules for a single product by going to the
+  product page and clicking the :guilabel:`Putaway Rules` smart button at the top of the page. If
+  the button isn't there at first glance, select the :guilabel:`More` button at the top right to
+  view additional configuration options.
 
-.. image:: putaway/putaw2.png
-   :align: center
+Once a putaway rule has been configured, the product it specifies will be automatically routed to
+the :guilabel:`Store to` location upon arriving in the :guilabel:`When product arrives in` location.
+The summary of internal product movements can be viewed by selecting :menuselection:`Reporting -->
+Product Moves` and enabling the :guilabel:`Internal` search filter on the :guilabel:`Filters`
+dropdown under the :guilabel:`search bar` at the top of the page.
 
```

Comment:
It could be useful to add a note or a tip here explaining what to do/look for if a putaway rule doesn't work as intended